CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add automatic codec degradation logic when CPU usage of software encoder is high or video encoding of encoder fails.
+
```

Comment:
@shi-su @hensmi-amazon If I'm not mistaken, this change is actually going out with 3.22.0, correct?

Reply:
Yes, good catch! this happens only like, 1/10 PRs 😅